recipe/build.sh

```diff
@@ -52,7 +48,7 @@ CMAKE_FLAGS+=" -DFFTW_THREADS_LIBRARY=${PREFIX}/lib/libfftw3f_threads${SHLIB_EXT
 # Build in subdirectory and install.
 mkdir -p build
 cd build
-cmake ${CMAKE_ARGS} ${CMAKE_FLAGS} ${SRC_DIR}
```

Comment:
Why? This is need for cross compiling

Reply:
It is messing with the CUDA detection somehow. See my comment above.

Reply:
You should then turn off `CMAKE_FIND_ROOT_PATH_MODE_INCLUDE` by setting `-DDCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=BOTH` instead to make cross compiling work.

Reply:
I was testing whether my hypothesis was correct or not. It was, and now I am trying to be a bit more selective with the CMAKE_ARGS modifications.

Reply:
Oh, sorry, didn't see that comment. Using your approach now.